hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieBaseRelation.scala

```diff
@@ -843,6 +843,15 @@ object HoodieBaseRelation extends SparkAdapterSupport {
       val hoodieConfig = new HoodieConfig()
       hoodieConfig.setValue(USE_NATIVE_HFILE_READER,
         options.getOrElse(USE_NATIVE_HFILE_READER.key(), USE_NATIVE_HFILE_READER.defaultValue().toString))
+
+      // Add HFile cache configurations from options
+      hoodieConfig.setValue(HFILE_BLOCK_CACHE_ENABLED,
+        options.getOrElse(HFILE_BLOCK_CACHE_ENABLED.key(), HFILE_BLOCK_CACHE_ENABLED.defaultValue().toString))
+      hoodieConfig.setValue(HFILE_BLOCK_CACHE_SIZE,
+        options.getOrElse(HFILE_BLOCK_CACHE_SIZE.key(), HFILE_BLOCK_CACHE_SIZE.defaultValue().toString))
+      hoodieConfig.setValue(HFILE_BLOCK_CACHE_TTL_MINUTES,
+        options.getOrElse(HFILE_BLOCK_CACHE_TTL_MINUTES.key(), HFILE_BLOCK_CACHE_TTL_MINUTES.defaultValue().toString))
```

Comment:
Extract this to a new util for reuse by different callers on `HoodieIOFactory.getIOFactory(storage).getReaderFactory(HoodieRecordType.AVRO).getFileReader`?

Reply:
Will move this to ConfigUtils.